src/databricks/labs/ucx/hive_metastore/catalog_schema.py

```diff
@@ -72,6 +75,16 @@ def _apply_from_legacy_table_acls(self) -> None:
                 continue
             logger.debug(f"Migrating acls on {grant.this_type_and_key()} using SQL query: {acl_migrate_sql}")
             self._backend.execute(acl_migrate_sql)
+        own_grants = [grant for grant in grants if grant.action_type == "OWN"]
+        for grant in own_grants:
+            own_acl_migrate_sql = grant.uc_grant_sql()
+            logger.debug(f"Migrating ownership on {grant.this_type_and_key()} using SQL query: {own_acl_migrate_sql}")
```

Comment:
this debug statement does not make sense logically here, move it after the if-continue

Reply:
Of course, the if continue was added in the review.